Build.fsx

```diff
@@ -1,4 +1,4 @@
-﻿#r @"packages/FAKE.4.19.0/tools/FakeLib.dll"
+﻿#r @"packages/FAKE.Core/tools/FakeLib.dll"
```

Comment:
I think a separate commit for deleting the old package (and its related documentation) is enough. No need for a new PR. 😉